pkg/domain/infra/abi/containers.go

```diff
@@ -509,3 +510,110 @@ func (ic *ContainerEngine) ContainerExec(ctx context.Context, nameOrId string, o
 	ec, err = terminal.ExecAttachCtr(ctx, ctr, options.Tty, options.Privileged, options.Envs, options.Cmd, options.User, options.WorkDir, &options.Streams, options.PreserveFDs, options.DetachKeys)
 	return define.TranslateExecErrorToExitCode(ec, err), err
 }
+
+func (ic *ContainerEngine) ContainerStart(ctx context.Context, namesOrIds []string, options entities.ContainerStartOptions) ([]*entities.ContainerStartReport, error) {
```

Comment:
Combining start and attach into one call in the bindings is going to be iffy for tunnel, but I don't know if we can do otherwise for ABI... Eek.

Reply:
on the tunnel side, that is going to be ok to have multiple binding calls ... i think we do this elsewhere.